spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,191 @@
+                expect(first_facility.name).to eq('JAMAICA')
+            end
+
+            it 'creates proper address for the NY DMV Facility instances' do
+                first_facility = @factory.create_facilities(@new_york_facilities).first
+
+                expect(first_facility.address).not_to eq(nil)
+                expect(first_facility.address).to be_a(String)
+                expect(first_facility.address).to eq('168-46 91ST AVE., 2ND FLR  JAMAICA NY 11432')
+            end
+
+            it 'creates proper phone number for the NY DMV Facility instances' do
+                first_facility = @factory.create_facilities(@new_york_facilities).first
+
+                expect(first_facility.phone).not_to eq(nil)
+                expect(first_facility.phone).to be_a(String)
+                expect(first_facility.phone).to eq('7189666155')
+            end
+
+            it 'creates proper daily hours for the NY DMV Facility instances' do
+                first_facility = @factory.create_facilities(@new_york_facilities).first
+
+                expect(first_facility.hours).not_to eq(nil)
+                expect(first_facility.hours).to be_a(String)
+                expect(first_facility.hours).to eq('Weekdays: 7:30 AM - 5:00 PM, Closed Saturdays and Sundays')
+            end
+        end
+
+        describe 'creating MO facilities' do
+            it 'can use data to create new MO DMV Facility instances' do
+                facilities = @factory.create_facilities(@missouri_facilities)
+
+                expect(facilities).to be_an_instance_of(Array)
+                expect(facilities.size).to be_an(Integer)
+                expect(facilities.first).to be_an_instance_of(Facility)
+                expect(facilities.last).to be_an_instance_of(Facility)
+            end
+
+            it 'creates proper name for the MO DMV Facility instances' do
+                first_facility = @factory.create_facilities(@missouri_facilities).first
+
+                expect(first_facility.name).not_to eq(nil)
+                expect(first_facility.name).to be_a(String)
+                expect(first_facility.name).to eq('NEVADA')
+            end
+
+            it 'creates proper address for the MO DMV Facility instances' do
+                first_facility = @factory.create_facilities(@missouri_facilities).first
+
+                expect(first_facility.address).not_to eq(nil)
+                expect(first_facility.address).to be_a(String)
+                expect(first_facility.address).to eq('409 E AUSTIN BLVD  NEVADA MO 64772')
+            end
+
+            it 'creates proper phone number for the MO DMV Facility instances' do
+                first_facility = @factory.create_facilities(@missouri_facilities).first
+
+                expect(first_facility.phone).not_to eq(nil)
+                expect(first_facility.phone).to be_a(String)
+                expect(first_facility.phone).to eq('(417) 448-5705')
```

Comment:
These tests are great -- very robust and organized. Because we're calling an API though, there is the possible risk that this data could change. For the purposes of this project, this is not a problem, but typically when we are handling external data, we won't want to make assertions about the actual values of the first facility in case those changes. It is sufficient to just test that the attribute is the correct datatype